base/assets/gm4/lang/en_us.json

```diff
@@ -67,6 +68,41 @@
     "block.gm4.ender_hopper": "Ender Hopper",
     "block.gm4.master_crafter": "Mastercraftsman's Bench",
     "block.gm4.treasure_chest": "Treasure Chest",
+    "block.gm4.liquid_tank.empty": "Empty Tank",
+    "block.gm4.liquid_tank.water": "Water Tank",
+    "block.gm4.liquid_tank.lava": "Lava Tank",
+    "block.gm4.liquid_tank.milk": "Milk Tank",
+    "block.gm4.liquid_tank.mushroom_stew": "Mushroom Stew Tank",
+    "block.gm4.liquid_tank.rabbit_stew": "Rabbit Stew Tank",
+    "block.gm4.liquid_tank.beetroot_soup": "Beetroot Soup Tank",
+    "block.gm4.liquid_tank.experience": "Experience Tank",
+    "block.gm4.liquid_tank.fire_resistance": "Fire Resistance Potion Tank",
+    "block.gm4.liquid_tank.floating": "Floating Potion Tank",
+    "block.gm4.liquid_tank.harming": "Harming Potion Tank",
+    "block.gm4.liquid_tank.healing": "Healing Potion Tank",
+    "block.gm4.liquid_tank.invisibility": "Invisibility Potion Tank",
+    "block.gm4.liquid_tank.leaping": "Leaping Potion Tank",
+    "block.gm4.liquid_tank.luck": "Luck Potion Tank",
+    "block.gm4.liquid_tank.night_vision": "Night Vision Potion Tank",
+    "block.gm4.liquid_tank.poison": "Poison Potion Tank",
+    "block.gm4.liquid_tank.regeneration": "Regeneration Potion Tank",
+    "block.gm4.liquid_tank.slow_falling": "Slow Falling Potion Tank",
+    "block.gm4.liquid_tank.slowness": "Slowness Potion Tank",
+    "block.gm4.liquid_tank.strength": "Strength Potion Tank",
+    "block.gm4.liquid_tank.strong_harming": "Strong Harming Potion Tank",
+    "block.gm4.liquid_tank.strong_healing": "Strong Healing Potion Tank",
+    "block.gm4.liquid_tank.strong_leaping": "Strong Leaping Potion Tank",
+    "block.gm4.liquid_tank.strong_poison": "Strong Poison Potion Tank",
+    "block.gm4.liquid_tank.strong_regeneration": "Strong Regeneration Potion Tank",
+    "block.gm4.liquid_tank.strong_slowness": "Strong Slowness Potion Tank",
+    "block.gm4.liquid_tank.strong_strength": "Strong Strength Potion Tank",
+    "block.gm4.liquid_tank.strong_swiftness": "Strong Swiftness Potion Tank",
+    "block.gm4.liquid_tank.strong_turtle_master": "Strong Turtle Master Potion Tank",
+    "block.gm4.liquid_tank.swiftness": "Swiftness Potion Tank",
+    "block.gm4.liquid_tank.turtle_master": "Turtle Master Potion Tank",
+    "block.gm4.liquid_tank.water_breathing": "Water Breathing Potion Tank",
+    "block.gm4.liquid_tank.weakness": "Weakness Potion Tank",
+    "block.gm4.liquid_tank.empty": "Empty Potion Tank",
```

Comment:
The block.gm4.liquid_tank.empty key is used twice. Perhaps this should be empty_potion?

Reply:
Accidently left that line in. Sorry